### ISSUE TITLE
Alchemy bug? + suggestion

### ISSUE THREAD

**Issue by iobiarg** [Azanor/thaumcraft-beta#1425]
hi there, been using and testing and having some issues with certain alchemy stuff: i can create crystals in the crucible for every aspect but not for pannus iter and fames (so far) the essence just stay in the crucible until it starts to go into the air, i can take all the other aspect with the crystals (lets see, from leather, i can take out bestia and stuff) but pannus stays in the crucible. Same with iter and fames... any clue what could be going on?I'm using tc6 beta24 pluss tc addons: thaumic basses, thaumic additions, thaumic pheripery and rustic+ thaumaturgy in case you know any of this could cause the bug.

Suggestion: it would be awesome eventually getting some sort of storage shelves (not bibliocraft pls) to put warded jars and actually be able to see the content, and extra compat with storage drawers too. I mean there should be a storage from this mod to get full compat... with tc6 woods and others

**Comment by wamc2017** [comment]
Hmm, TC6 doesn't add Iter, Fames or Pannus, those are added by Thaumic Bases. If all other aspects are fine, you may should report to TBs issue tracker: https://github.com/Rumaruka/Thaumic-Bases-Unofficial/issues

**Comment by iobiarg** [comment]
Thanks Wamc2017. Sorry for bothering... Still try to take into account my suggestion pls.

**Comment by Azanor** [comment]
Yes, custom aspects will need to add their own crafting recipes depending on how and where they add the aspects.
